examples/vae/train.py

```diff
@@ -11,90 +11,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Traininga and evaluation logic."""
 
-from absl import app
-from absl import flags
+from absl import logging
 from flax import linen as nn
 from flax.training import train_state
-import jax.numpy as jnp
+import input_pipeline
 import jax
 from jax import random
-import numpy as np
+import jax.numpy as jnp
+import models
 import optax
-import tensorflow as tf
-import tensorflow_datasets as tfds
-
 import utils as vae_utils
```

Comment:
nit insert empty space to separate local imports from other library imports.

Reply:
Fixed